xml/depl_nodes.xml

```diff
@@ -54,6 +54,12 @@
  </para>
  <orderedlist spacing="normal">
   <listitem>
+<!-- Designate -->
+   <para>
+    <xref linkend="sec.depl.ostack.designate" xrefstyle="select:title nopage"/>
+   </para>
+  </listitem>
+  <listitem>
 <!-- Pacemaker -->
    <para>
     <xref linkend="sec-depl-ostack-pacemaker" xrefstyle="select:title nopage"/>
```

Comment:
```suggestion
    <xref linkend="sec.depl.ostack.pacemaker" xrefstyle="select:title nopage"/>
```

Reply:
good point!

Reply:
No, please use the dashes from now on. We had to change the format with the change to the new server and the period's will break the build :)

Reply:
Which is to say, if the xref link is with the periods, it should also be changed to dashes.

Reply:
Done.